testing/simulator/src/eth1_sim.rs

```diff
@@ -17,10 +17,7 @@ pub fn run_eth1_sim(matches: &ArgMatches) -> Result<(), String> {
         .expect("missing validators_per_node default");
     let speed_up_factor =
         value_t!(matches, "speed_up_factor", u64).expect("missing speed_up_factor default");
-    let mut end_after_checks = true;
-    if matches.is_present("end_after_checks") {
-        end_after_checks = false;
-    }
+    let end_after_checks = !matches.is_present("end_after_checks");
```

Comment:
This seems wrong (both before and after)

Reply:
We could fix it by renaming the flag to `continue_after_checks`

Reply:
Yeah, if this is a bug, maybe fix it after the pull request got merged (since it has nothing to do with the clippy fixes)?

Reply:
Yeah, true. I'll make an issue